templates/bot/ts/workflow/README.md

```diff
@@ -129,7 +129,7 @@ export class DoSomethingActionHandler {
     triggerVerb = "doSomething";
 
     async handleActionInvoked(context, message) { 
```

Comment:
How about change the parameter name to `actionData` here (to keep consistency with the action handler API) ?

Reply:
Done!